<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Authentication</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="log_in">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="log_in_login"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="log_in_password">
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="signInButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Sign In</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="signUpButton">
              <property name="text">
               <string>Sign Up</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registarte">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="registrate_login"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="registrate_password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Confirm Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="registrate_confirm">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="registrateButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Registarte</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>log_in_login</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RegisterWindow</receiver>
   <slot>passwordOrLoginIsEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>log_in_password</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RegisterWindow</receiver>
   <slot>passwordOrLoginIsEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registrate_login</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RegisterWindow</receiver>
   <slot>registrationDataIsEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registrate_password</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RegisterWindow</receiver>
   <slot>registrationDataIsEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registrate_confirm</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RegisterWindow</receiver>
   <slot>registrationDataIsEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signInButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterWindow</receiver>
   <slot>connection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registrateButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisterWindow</receiver>
   <slot>registrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>passwordOrLoginIsEdited()</slot>
  <slot>registrationDataIsEdited()</slot>
  <slot>connection()</slot>
  <slot>registrate()</slot>
 </slots>
</ui>
